src/ARCTokenization/Tokenization.fs

```diff
@@ -158,3 +158,87 @@ module Tokenization =
                 at.Columns
                 |> Array.map CompositeColumn.tokenize
                 |> List.ofArray
+
+    module SpecificTokens =
+        let matchPathToCVTerms (rootPath:string) =
+            let root =  System.Uri(rootPath)
+
+            let segmentor = 
+                root.Segments.[0].ToCharArray().[0]
+
+            let arcName = 
+                root.Segments
+                |>Array.last
+                |>fun x -> 
+                    x.Trim(segmentor)  
+
+            let trimmSegments (p:string) = 
+                p,
+                p.Split([|arcName|],System.StringSplitOptions.RemoveEmptyEntries)
+                |>Array.tail
+                |> fun x -> x.[0].Trim(segmentor).Split segmentor
+            
+            let getC (p: string) (segments: string array) = 
+                match Array.length segments with
+                | 1 -> 
+                    if Array.contains "studies" segments then
+                        p,StructuralOntology.AFSO.``Studies Directory``
+                    elif Array.contains "assays" segments then
+                        p,StructuralOntology.AFSO.``Assays Directory``
+                    elif Array.contains "runs" segments then
+                        p,StructuralOntology.AFSO.``Runs Directory``
+                    elif Array.contains "workflows" segments then
+                        p,StructuralOntology.AFSO.``Workflows Directory``
+                    else
+                        p,StructuralOntology.AFSO.``Directory Path``
+                | 2 -> 
+                    if Array.contains "studies" segments then
```

Comment:
Also note that ARCtrl provides literal strings for most of these well-defined strings, so we should use those (e.g., reference them in the `Globals` namespace and use those references here)